src/components/UNSTABLE__Shell/NotificationsPopover/index.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file Notifications popover.
+ * @copyright IBM Security 2020
+ */
+
+import Close20 from '@carbon/icons-react/lib/close/20';
+
+import { func, node, string } from 'prop-types';
+import React from 'react';
+
+import theme from '../../../globals/theme';
+
+import Icon from '../../Icon';
+import ScrollGradient from '../../ScrollGradient';
+
+import PopoverContent from '../PopoverContent';
+import PopoverHeader from '../PopoverHeader';
+import PopoverTitle from '../PopoverTitle';
+
+import namespace from '../Popover';
+
+function NotificationsPopover({
+  children,
+  iconDescription,
+  label,
+  onClear,
+  title,
+  ...other
+}) {
+  return (
+    <>
+      <PopoverHeader>
+        <PopoverTitle>{title}</PopoverTitle>
+      </PopoverHeader>
+      <PopoverContent {...other}>
+        {label && <span className={`${namespace}__label`}>{label}</span>}
+        {onClear && (
+          <button
+            className={`${namespace}__button`}
+            aria-label={iconDescription}
+            onClick={onClear}
+          >
+            <Icon renderIcon={Close20} />
+          </button>
```

Comment:
Were you able to use a Carbon `Button` element with `renderIcon` instead?

Reply:
Good point! I'll use the `IconButton` here